movie_bet_bot/utils/constants.py

```diff
@@ -139,8 +149,8 @@
             {members}
         </div>
         <hr>
```

Comment:
Would like to remove the break for updates/images where no new films were watched, but this is fine for now